server.js

```diff
@@ -1,38 +1,34 @@
 // TASK 1: Setting up server
 
 
-
```

Comment:
The constant HOSTNAME should be in uppercase to denote that it is a constant value.